pkg/util/kubernetes/controller/history/controller_history.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2024 The OpenYurt Authors.
```

Comment:
how about add K8s license: `Copyright 20xx The Kubernetes Authors`

Reply:
ok, have replaced openyurt license to `Copyright 2017 The Kubernetes Authors`